packages/shared/src/aws/tile.metadata.base.ts

```diff
@@ -190,6 +193,55 @@ export class TileMetadataTableBase {
         return DynamoDB.Converter.unmarshall(item.Item) as T;
     }
 
+    /**
+     * Iterate over the entire table yielding each batch of items
+
+     * @param TableName the name of the table to interate over
+     */
+    async *scan(TableName: string): AsyncGenerator<DynamoDB.Types.ItemList, null, void> {
+        let exclusiveStartKey: DynamoDB.Key | undefined = undefined;
+        while (true) {
+            const items: DynamoDB.Types.ScanOutput = await this.dynamo
+                .scan({
+                    TableName,
+                    ExclusiveStartKey: exclusiveStartKey,
+                })
+                .promise();
+
+            if (items.Items == null) break;
+
+            yield items.Items;
```

Comment:
Would be better to yield these one by one, and include the type of the base object.

This should also do a migrate?

Reply:
much more useful to me the way I have written it

Reply:
No longer needed as is because `export --full` removed